contracts/duckies/nft/DucklingFamilyGame/DucklingFamilyGameV1.sol

```diff
@@ -0,0 +1,245 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.18;
+
+import '@openzeppelin/contracts-upgradeable/utils/CountersUpgradeable.sol';
+import '@openzeppelin/contracts-upgradeable/utils/StringsUpgradeable.sol';
+import '@openzeppelin/contracts-upgradeable/utils/cryptography/ECDSAUpgradeable.sol';
+
+import '../../../interfaces/IVoucher.sol';
+import '../RandomUpgradeable.sol';
+import '../Gene.sol';
+
+contract DucklingFamily is IVoucher, RandomUpgradeable {
+	using CountersUpgradeable for CountersUpgradeable.Counter;
+	using Gene for uint256;
+
+	// ------- IVoucher -------
+
+	enum VouhcherActions {
+		MintPacks,
+		MeldFlocks
```

Comment:
MeldFlock (it's a group of birds)

Reply:
f1e7948f00ec9ab95587b20e61774e67a8048737